fiftyone/core/threed/miris_stream.py

```
@@ -0,0 +1,60 @@
+    ):
+        """Initializes a :class:`MirisStream`."""
+        super().__init__(
+            name=name,
+            visible=visible,
+            position=position,
+            scale=scale,
+            quaternion=quaternion,
+        )
+
+        if not asset_uuid:
+            raise ValueError("MirisStream requires an asset_uuid")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)